server/scripts/createTwimlApp.js

```diff
@@ -33,9 +32,15 @@ async function getNgrokTunnel() {
 async function main() {
   const tunnel = await getNgrokTunnel();
 
-  const twimlApp = await client
-    .applications(TWIML_APP_SID)
-    .update({ voiceUrl: `${tunnel.public_url}/twiml` });
+  const twimlApp = await client.applications.create({
+    voiceUrl: `${tunnel.public_url}/twiml`,
+  });
+
+  fs.appendFile('../.env', `CURRENT_TWIML_APP_SID=${twimlApp.sid}\n`, (err) => {
+    if (err) {
+      console.log(err);
+    }
```

Comment:
If this occurs, it will pretty much fail "silently". We'll see a log sure, but the CI will continue because the exit value of the script is still zero.

Reply:
Take a look at this https://nodejs.org/api/process.html#processexitcode

Reply:
For posix compliance, we should be returning an exit-code of non-zero for the script to be considered "failed", typically just `1`.

Reply:
Oh apparently `process.exit` is not preferred and `process.exitCode` is, but same idea.